src/systems/lift_drag/LiftDrag.cc

```diff
@@ -281,6 +293,12 @@ void LiftDragPrivate::Update(EntityComponentManager &_ecm)
   // rotate forward and upward vectors into world frame
   const auto forwardI = pose.Rot().RotateVector(this->forward);
 
+  if (forwardI.Dot(vel) <= 0.0){
```

Comment:
I don't follow this. Why? Is returning the right behavior here?

Reply:
If relative velocity of the vehicle forms a negative dot product with the forward vector then no lift can be generated and it does not need to be calculated.

Reply:
Noted! Thanks. Makes sense.